Snippets/Gateway/Gateway_2/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "Microsoft.ServiceFabric.Services": "2.5.216",
```

Comment:
https://github.com/Particular/docs.particular.net/pull/2602/files#diff-290f49db3155932839f03b1f8c0b0efa uses `2.*`. Should we align?